cpp/include/raft/neighbors/detail/refine.cuh

```diff
@@ -129,10 +130,13 @@ void refine_device(raft::resources const& handle,
            queries.data_handle(),
            fake_coarse_idx.data(),
            static_cast<uint32_t>(n_queries),
+           0,
            refinement_index.metric(),
            1,
            k,
            raft::distance::is_min_close(metric),
+           // TODO: add the filtering support
```

Comment:
I think, we can remove this todo because `refine` does not need filtering support (it takes already filtered results).

Am I right, @cjnolet?

Reply:
That sounds right: if the neighbor candidates are already filtered, then we do not need any extra filtering in refine.